src_Testbench/SoC/SoC_Top.bsv

```diff
@@ -90,15 +91,9 @@ import Debug_Module     :: *;
 // The outermost interface of the SoC
 
 interface SoC_Top_IFC;
-   // Set core's verbosity
-   method Action  set_verbosity (Bit #(4)  verbosity, Bit #(64)  logdelay);
 
 `ifdef INCLUDE_GDB_CONTROL
-   // DMI (Debug Module Interface) facing remote debugger
-   interface DMI dmi;
-
-   // Non-Debug-Module Reset (reset all except DM)
-   interface Client #(Bool, Bool) ndm_reset_client;
+  interface AXI4Lite_Slave #(7, 32, 0, 0, 0, 0, 0) debug_subordinate;
```

Comment:
What are 7 and 32 here?

Reply:
they are is the address width and the data width of the dmi interface (at least they should be)